frontend/src/metabase/components/Toaster/Toaster.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect, HTMLAttributes } from "react";
+import {
+  ToasterContainer,
+  ToasterMessage,
+  ToasterButton,
+} from "./Toaster.styled";
+import Icon from "metabase/components/Icon";
+import { color } from "metabase/lib/colors";
+
+export interface ToasterProps extends HTMLAttributes<HTMLAnchorElement> {
+  message: string;
+  show: boolean;
```

Comment:
WDYT about using a name like `isShown` or `isVisible` to indicate it's a boolean and not a function?